Cubical/Data/Empty/Base.agda

```diff
@@ -10,3 +10,6 @@ rec ()
 
 elim : ∀ {ℓ} {A : ⊥ → Type ℓ} → (x : ⊥) → A x
 elim ()
+
+nope : ∀ {ℓ} (A : Type ℓ) → Type ℓ
```

Comment:
I think this should be `rec`?

Reply:
wuups... this is just me goofing around, I didn't mean to push this stuff

Reply:
I think it's a fine addition if it's called rec. :-)

Do you know `git gui`? It lets you select exactly what goes into a commit. I use it *all the time*. That way you can much easier get an overview of what to include in a commit. You can also easily commit chunks or even lines of different files separately.

Reply:
Typically I would fix all of the comments on my PR like you have done, but then commit the changes in smaller chunks corresponding to every fix using `git gui`. That way it's much easier to revert a commit in case the changes turn out to be unnecessary.

Reply:
> I think it's a fine addition if it's called rec. :-)

Ah, but that's already in there. `nope` was just supposed to be negation.

> Do you know `git gui`? It lets you select exactly what goes into a commit. I use it _all the time_. That way you can much easier get an overview of what to include in a commit. You can also easily commit chunks or even lines of different files separately.

Oh yeah, I should start using that for sure...

Reply:
Oops, I didn't look closely enough at what `nope` was... Then please just remove it :-) We already have negation in Relation.Unary